packages/sdk-metrics/src/Instruments.ts

```diff
@@ -36,7 +37,6 @@ import {
   AsyncWritableMetricStorage,
   WritableMetricStorage,
 } from './state/WritableMetricStorage';
-import { Gauge } from './types';
```

Comment:
We should keep using `Gauge` from `./types` as it could still happen that `@opentelemetry/api@1.3.0` which does not have the `Gauge` type.